.github/scripts/parse_labels.py

```diff
@@ -17,9 +17,10 @@ class ParsedLabels(TypedDict):
 
 
 def parse_labels(docker_compose: dict, service_name: str) -> Dict[str, ParsedLabels]:
-    """Given the docker-compose in dict/json form, return the label information."""
+    """Given the docker-compose in dict/json form, return the label information and label order."""
     services = docker_compose["services"]
     parsed_labels = {}
+    env_order = 0
 
     for service in services:
```

Comment:
En python se puede usar ```enumerate``` para obtener el índice en un bucle sin tener que crear una variable auxiliar fuera del mismo:

```suggestion
    for env_order, service in enumerate(services):
```

¿Te animas a cambiarlo o lo dejo así?

Reply:
Lo que yo quiero hacer no es enumerar los servicios, quiero contar los grupos de labels de cada variable de entorno.
Por eso, cada vez que un label no se encuentra en `service_labels["envs"]`, se suma uno al contador.
Si aplicara ese cambio, el orden de cada label no se aplicaría bien.

Reply:
Ok, perdona @aitorcas23, con el diff de la interfaz web no había visto todos los "if"s intermedios que hay en el código, sólo me había fijado en lo que se muestra aquí. Tienes razón 😉